### PR TITLE
Create vignette to illustrate custom CSS

### LINKED ISSUE #48: Add example of using class option of rank_list
I have no idea about css classes, but I need change the style of my rank_list, e.g. change the background colour of the list items. 

An example might be very helpful here.

### PR DESCRIPTION
**Pull request by andrie**
Closes #48

### CODE REVIEW

File: vignettes/using_custom_css.Rmd [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+## Source code
+
+```{r echo=FALSE, cache=FALSE}
+knitr::read_chunk(
+  system.file("shiny-examples/custom_css/app.R", package = "sortable")
+)
+```
+
+```{r custom-css-app, eval=FALSE}
```
Comment (schloerke) [inline vignettes/using_custom_css.Rmd]: What is this for?

File: vignettes/using_custom_css.Rmd [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+For example, to modify the `custom-sortable` class using a `shiny` tag, use:
+
+```r
+tags$style(
+        HTML("
+             .rank-list-container.custom-sortable {background-color: #8A8;}
+             .custom-sortable .rank-list-item {background-color: #BDB;}
+
+             ")
+        )
```
Comment (schloerke) [inline vignettes/using_custom_css.Rmd]: Can these be indented closer to the left?

File: inst/shiny-examples/custom_css/app.R [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+          "three",
+          htmltools::tags$div(
+            htmltools::em("Complex"), " html tag without a name"
+          ),
+          "five" = htmltools::tags$div(
+            htmltools::em("Complex"), " html tag with name: 'five'"
+          )
+        ),
+        input_id = "rank_list_1",
+        class = c("default-sortable custom-sortable") # add custom style
```
Comment (schloerke) [inline inst/shiny-examples/custom_css/app.R]: Can this be done as `c("default-sortable", "custom-sortable")`? We auto collapse the class values.

**Comment by schloerke** [comment]
Could you also sign the CLA thing? While we're the main authors, it makes everything easier to work with. 😄  Thank you!

File: vignettes/using_custom_css.Rmd (hunk 1 above)
Comment (andrie) [reply vignettes/using_custom_css.Rmd]: This some R markdown magic to include a chunk of code from a different document.  The effect is to include the code from the app directly in the vignette, but without evaluation.

In short: we get the running app from ShinyApps.io, and the code below this for inspection.

It's the same mechanism we use to create https://rstudio.github.io/sortable/articles/novel_solutions.html

File: vignettes/using_custom_css.Rmd (hunk 2 above)
Comment (andrie) [reply vignettes/using_custom_css.Rmd]: I'm fixing.

File: inst/shiny-examples/custom_css/app.R (hunk 3 above)
Comment (andrie) [reply inst/shiny-examples/custom_css/app.R]: Nice.  Will do.

**Pull request merged**